taui/src/components/neighborhood-details.js

```diff
@@ -72,13 +71,13 @@ class NeighborhoodDetails extends PureComponent<Props> {
 
     return (
       <div className='neighborhood-details__trip'>
-        {bestJourney && <span className='neighborhood-details__route'>{t('Units.About')}&nbsp;
-          {roundedTripTime}&nbsp;
-          {t('Units.Mins')}&nbsp;
-          <ModesList segments={bestJourney} />&nbsp;
-          {t('NeighborhoodDetails.FromOrigin')}&nbsp;
-          {currentDestination && t('TripPurpose.' + currentDestination.purpose).toLowerCase()}
-        </span>}
+        {bestJourney && <span className='neighborhood-details__route'>
+          <strong>{`${t('NeighborhoodDetails.TravelTime')}: `}</strong>
```

Comment:
These constructions could have some punctuation removed by structuring them as:
```suggestion
          <strong>{t('NeighborhoodDetails.TravelTime')}: </strong>
```
To my eye it feels easier to parse because it removes a layer of nesting this way.